relayer/application.ts

```diff
@@ -196,26 +235,44 @@ export class RelayerApp<ContextT extends Context> {
     }
   }
 
+  /**
+   * Gives you a Chain router so you can add middleware on an address.
+   * @example:
+   * ```
+   * relayerApp.chain(CHAIN_ID_ETH).address("0x0001234abcdef...", middleware1, middleware2);
+   * ```
+   *
+   * @param chainId
+   */
   chain(chainId: ChainId): ChainRouter<ContextT> {
     if (!this.chainRouters[chainId]) {
       this.chainRouters[chainId] = new ChainRouter(chainId);
     }
     return this.chainRouters[chainId]!;
   }
 
+  /**
```

Comment:
Meta: should middlewares be able to extend the app? This seems like a good convenience function that should become available iff the token bridge middleware is being used. Extending the app seems like an interesting extension point.